bing.com.BingWebMasterDPOOV1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOOV2",
+  "serviceName": "Domain proof of ownership with Bing Webmaster services",
+  "version": 1,
```

Comment:
If you'll change the serviceId back to BingWebMasterDPOO (see comment above) please bump the version here.

Reply:
Sure